File: Examples/CSharp/WorkingWithTasks/ViewSplitTasks.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithTasks
{
    using System;

    internal class ViewSplitTasks
    {
        public static void Run()
        {
            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod());

            //ExStart:ViewSplitTasks
            //ExFor: Task.SplitParts
            //ExSummary: Shows how to display task's split parts.
            var project = new Project(dataDir + "ViewSplitTasks.mpp");
            
            // Access task 
            var splitTask = project.RootTask.Children.GetById(4);

            // Display split parts of task
            var collection = splitTask.SplitParts;
            foreach (var splitPart in collection)
            {
                Console.WriteLine("Index: " + splitPart.Index + "\nStart: " + splitPart.Start + "\nFinish: " + splitPart.Finish + "\n");
            }
            //ExEnd:ViewSplitTasks
        }
    }
}

File: Examples/VisualBasic/WorkingWithTasks/ViewSplitTasks.vb

Namespace WorkingWithTasks
    Public Class ViewSplitTasks
        Public Shared Sub Run()
            ' ExStart:ViewSplitTasks
            ' Create project instance
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            Dim project1 As New Project(dataDir & "project.mpp")

            ' Access Task
            Dim splitTask As Task = project1.RootTask.Children.GetById(4)

            ' Display split parts of task
            Dim collection As SplitPartCollection = splitTask.SplitParts
            For Each splitPart As SplitPart In collection
                Console.WriteLine("Index: " & splitPart.Index & Constants.vbLf & "Start: " & splitPart.Start & Constants.vbLf & "Finish: " & splitPart.Finish & Constants.vbLf)
            Next splitPart
            ' ExEnd:ViewSplitTasks
        End Sub
    End Class

End Namespace
